maro/simulator/scenarios/vm_scheduling/common.py

```diff
@@ -106,3 +106,17 @@ def __repr__(self):
 
     def __str__(self):
         return f'Latency(Agent={self.due_to_agent}, Resource={self.due_to_resource})'
+
+
+class PmState(IntEnum):
+    """PM oversubscription state, includes empty, oversubscribable, non-oversubscribable."""
+    NON_OVERSUBSCRIBABLE = -1
+    EMPTY = 0
+    OVERSUBSCRIBABLE = 1
+
+
+class Sku:
```

Comment:
may also include the core number, memory capacity, price, ...?

confirm the SKU definition first

Reply:
Wondering should we really need SKU class? Or maybe we can easily use a dictionary to keep the sku information.
```python
{ 0: {
          'amount': 50,
          'price': 0,
          'cpu': 1,
          ...
     },
 1:  {
       ...
     }
}
```